src/viewProviders/topics.test.ts

```diff
@@ -76,7 +76,7 @@ describe("TopicViewProvider helper functions", () => {
     assert.equal(schemas.length, 0);
   });
 
-  it("loadTopicSchemas() should return schemas for CCloud Kafka topics when available", async () => {
+  it.skip("loadTopicSchemas() should return schemas for CCloud Kafka topics when available", async () => {
```

Comment:
Paired on this and click-tested the behavior, which is fine -- the main thing we'll need in a follow-up branch is preloading the ResourceLoader in the test environment. 👍